core/src/apps/bitcoin/get_address.py

```diff
@@ -107,6 +114,15 @@ async def get_address(msg: GetAddress, keychain: Keychain, coin: CoinInfo) -> Ad
 
             await confirm_multisig_warning()
 
+            # An addresss that uses different derivation paths for different xpubs could be difficult to discover
```

Comment:
typo "addresss"

Reply:
Also, how about adding a bit more to the rationale, along the lines that malicious software on the host computer could use this to execute a ransom attack.